target/linux/realtek/dts-5.10/rtl931x.dtsi

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
This file should have 0644 permissions, not 0755.

Reply:
Fixed.